lib/management/agentClient.js

```diff
@@ -0,0 +1,16 @@
+const managementAgentMessageType = {
+    /** Message that contains a configuration overlay */
+    NEW_OVERLAY_VERSION: 1,
+    /** Message that contains the loaded overlay */
+    LOADED_OVERLAY: 2,
```

Comment:
Why is there a different message type `LOADED_OVERLAY`? I think only one message type to push overlays would be clearer.